frontend/src/scenes/dashboard/NewDashboardModal.tsx

```diff
@@ -1,21 +1,153 @@
+import { newDashboardLogic } from 'scenes/dashboard/newDashboardLogic'
+import { LemonModal } from 'lib/lemon-ui/LemonModal'
+import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
+import { FEATURE_FLAGS } from 'lib/constants'
+import { dashboardTemplatesLogic } from 'scenes/dashboard/dashboards/templates/dashboardTemplatesLogic'
+import { DashboardTemplateVariables } from './DashboardTemplateVariables'
+import { LemonButton } from '@posthog/lemon-ui'
+import { dashboardTemplateVariablesLogic } from './dashboardTemplateVariablesLogic'
+import { DashboardTemplateType } from '~/types'
+import { useState } from 'react'
+
 import { Field } from 'lib/forms/Field'
-import { LemonButton } from 'lib/lemon-ui/LemonButton'
 import { AvailableFeature } from '~/types'
 import { LemonSelect } from 'lib/lemon-ui/LemonSelect'
 import { PayGateMini } from 'lib/components/PayGateMini/PayGateMini'
-import { newDashboardLogic } from 'scenes/dashboard/newDashboardLogic'
 import { LemonInput } from 'lib/lemon-ui/LemonInput/LemonInput'
 import { LemonTextArea } from 'lib/lemon-ui/LemonTextArea/LemonTextArea'
 import { DASHBOARD_RESTRICTION_OPTIONS } from './DashboardCollaborators'
-import { LemonModal } from 'lib/lemon-ui/LemonModal'
 import { Form } from 'kea-forms'
-import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
-import { FEATURE_FLAGS } from 'lib/constants'
 import { userLogic } from 'scenes/userLogic'
-import { dashboardTemplatesLogic } from 'scenes/dashboard/dashboards/templates/dashboardTemplatesLogic'
+import { pluralize } from 'lib/utils'
+import { getSeriesColor } from 'lib/colors'
 
-export function NewDashboardModal(): JSX.Element {
+import './NewDashboardModal.scss'
+
+function FallbackCoverImage({ src, alt, index }: { src: string | undefined; alt: string; index: number }): JSX.Element {
+    const [hasError, setHasError] = useState(false)
+
+    const handleImageError = (): void => {
+        setHasError(true)
+    }
+
+    return (
+        <>
+            {hasError || !src ? (
+                <div
+                    className="w-full h-full"
+                    // dynamic color based on index
+                    // eslint-disable-next-line react/forbid-dom-props
+                    style={{
+                        background: getSeriesColor(index),
+                    }}
+                />
+            ) : (
+                <img
+                    className="w-full h-full object-cover object-center"
+                    src={src}
+                    alt={alt}
+                    onError={handleImageError}
+                />
+            )}
+        </>
+    )
+}
+
+function TemplateItem({
+    template,
+    onClick,
+    index,
+}: {
+    template: Pick<DashboardTemplateType, 'template_name' | 'dashboard_description' | 'image_url'>
+    onClick: () => void
+    index: number
+}): JSX.Element {
+    return (
+        <div className="cursor-pointer border-2 rounded TemplateItem" onClick={onClick}>
```

Comment:
Just one tiny thing I noticed: it'd be more typical for this to have a 1px border, not 2px. So just `border` instead of `border-2` to align this with other "cards" in our UI